src/utils/valuesConfig.ts

```diff
@@ -0,0 +1,59 @@
+import { DataClass, DataItem } from 'scrivito'
```

Comment:
I would rename this file to `dataValuesConfig.ts` to make it clearer, which values are ment.